deploy/roles/spark/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+- name: Create service account for Spark
+  user: name={{ spark_user }}
```

Comment:
@olivermel Sorry to nitpick, but could you use the same style as the other roles in Aurora?  So like this:
user:
  name: "{{ spark_user }}"
  system: yes
  etc...

Thanks!